app/models/performance_report.rb

```diff
@@ -55,12 +58,32 @@ def clients(zurich = false)
 
   def assignments(zurich = false)
     assignments = Assignment.created_before(period_end)
-    assignments = assignments.zurich if zurich
+    assignments = zurich ? assignments.zurich : assignments.not_zurich
+    assignments = extern ? assignments.external : assignments.internal
     {
       ended: assignments.end_within(period_start..period_end).count,
       new: assignments.start_within(period_start..period_end).count,
       active: assignments.active_between(period_start, period_end).count,
       total: assignments.count
     }
   end
+
+  def group_offers
+    group_offers = GroupOffer.created_before(period_end)
+    active = group_offers.map do |group_offer|
+      group_offer.has_active_group_assignments_between?(period_start, period_end)
+    end
+    ended = group_offers.map do |group_offer|
+      group_offer.all_group_assignments_ended_within?(period_start..period_end)
+    end
+    new_group_offers = group_offers.map do |group_offer|
+      group_offer.all_group_assignments_started_within?(period_start..period_end)
+    end
+    {
+      active: active.reject(&:blank?).size,
```

Comment:
Try `.compact` instead of `.reject(&:blank)`, should work the same if the blank values are `nil`

Reply:
but they are not nil, they're false

Reply:
nope, I just checked in pry:
```ruby
[false, false, true].compact # => [ [0] false, [1] false, [2] true ]
```

Reply:
but as I just discovered, `.reject(&:blank?)` only works in rails, not in plain ruby.

Reply:
but maybe 
```ruby
active.grep(true)
```
?